pkg/volume/flocker/plugin.go

```diff
@@ -113,6 +113,10 @@ type flockerBuilder struct {
 	readOnly bool
 }
 
+func (_ *flockerBuilder) SupportsOwnershipManagement() bool {
+	return false
```

Comment:
I am going to assume this is `false` for now.  We can enable it later once @agonzalezro has a chance to catch up with the work and make a call.